packages/frontend-2/components/dashboard/Sidebar.vue

```diff
@@ -162,19 +172,45 @@ import {
 } from '~/lib/common/helpers/route'
 import { useRoute } from 'vue-router'
 import { useActiveUser } from '~~/lib/auth/composables/activeUser'
-import { useNavigation } from '~~/lib/navigation/composables/navigation'
 import { useMixpanel } from '~~/lib/core/composables/mp'
+import { useActiveWorkspace } from '~/lib/user/composables/activeWorkspace'
+import { graphql } from '~/lib/common/generated/gql'
+import { useQuery } from '@vue/apollo-composable'
+
+const dashboardSidebarQuery = graphql(`
```

Comment:
isn't this more or less the same query thats already invoked in `useActiveWorkspace()`? you could just make useActiveWorkspace() return the id and role as well

Reply:
There will be some other information we need aswell in the sidebar, such as plan name and status to conditionally render some items, however that is not in this branch yet. It felt a bit overkill to always fetch that info